### PR TITLE
Feat/organizational widgets

### PR DESCRIPTION
**Pull request by DaniPoot**
- Create components for badge assignments, role assignments, and payouts for the organizational page
- Fix the vote list
- Fix the validation to show the correct color on voting bars
- Fix archive status

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -21,7 +21,7 @@ DGRAPH_ROOT_HASH=52a7ff82bd6f53b31285e97d6806d886eefb650e79754784e9d923d3df347c9
 
 GRAPHQL_URI=https://alpha-stts.tekit.io/graphql
 
-TELOS_ENDPOINTS=https://testnet.telos.caleos.io
+TELOS_ENDPOINTS=https://telos.caleos.io
```
Comment (jmgayosso) [inline .env.example:24]: why endpont  was changed???

File: src/pages/dho/Organizational.vue [hunk 2]
```diff
@@ -91,6 +94,60 @@ export default {
...
+          return {
+            title: assignment.details_title_s,
+            description: assignment.details_description_s,
+            docId: assignment.docId
+          }
+        })
+      },
+      variables () {
+        return {
+          daoName: this.selectedDao.name
```
Comment (jmgayosso) [inline src/pages/dho/Organizational.vue:147]: We should pass params to get just the last 3 items for each proposal type

Comment (DaniPoot) [reply src/pages/dho/Organizational.vue:147]: I'm going to add the variable

File: .env.example (hunk 1 above)
Comment (DaniPoot) [reply .env.example:24]: I think is an error because I made a mistake merging master. I didn't see this change I'm going to restore that

**Review by jmgayosso — APPROVED** [review]

**Pull request merged**